tests/cbmc/proofs/s2n_stuffer_read_hex/s2n_stuffer_read_hex_harness.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#include <assert.h>
+#include <cbmc_proof/cbmc_utils.h>
+#include <cbmc_proof/make_common_datastructures.h>
+
+#include "api/s2n.h"
+#include "stuffer/s2n_stuffer.h"
+
+void s2n_stuffer_read_hex_harness()
+{
+    nondet_s2n_mem_init();
+
+    struct s2n_stuffer *bytes_out = cbmc_allocate_s2n_stuffer();
+    __CPROVER_assume(s2n_result_is_ok(s2n_stuffer_validate(bytes_out)));
+
+    struct s2n_blob *hex_in = cbmc_allocate_s2n_blob();
+    __CPROVER_assume(s2n_result_is_ok(s2n_blob_validate(hex_in)));
+    __CPROVER_assume(s2n_blob_is_bounded(hex_in, MAX_BLOB_SIZE));
+
+    struct s2n_stuffer old_bytes_out = *bytes_out;
+    struct store_byte_from_buffer old_bytes_out_byte = { 0 };
```

Comment:
I don't really see any issues with this proof, but it is really hard to read. I think there's something going on with the naming here. "old_bytes_out_byte" just doesn't make much sense to me. Maybe you could call the stuffer just "output" and the blob "hex_input" and then you can reserve "byte" for when you're actually storing one byte.